Sources/FluentMongoDriver/MongoDatabase.swift

```diff
@@ -0,0 +1,478 @@
+import FluentKit
+import MongoKitten
+import MongoCore
+
+struct _MongoDB: DatabaseDriver, MongoDB {
+    func makeDatabase(with context: DatabaseContext) -> Database { self }
+    
+    func shutdown() { cluster.disconnect() }
+    
+    let cluster: MongoCluster
+    public let raw: MongoDatabase
+    let context: DatabaseContext
+    
+    init(cluster: MongoCluster, databaseName: String) {
+        self.cluster = cluster
+        self.raw = cluster[databaseName]
+        self.context = DatabaseContext(
+            configuration: .init(),
+            logger: Logger.defaultMongoCore,
+            eventLoop: cluster.eventLoop
+        )
+    }
+}
+
+public protocol MongoDB {
+    var raw: MongoDatabase { get }
```

Comment:
The naming is a bit unfortunate here. Is `MongoDatabase` a protocol?

Reply:
MongoDatabase is a class in MongoKitten.